src/stress/memory/oom.spec.ts

```diff
@@ -0,0 +1,68 @@
+export const description = `
+Stress tests covering robustness when available VRAM is exhausted.
+`;
+
+import { makeTestGroup } from '../../common/framework/test_group.js';
+import { GPUTest } from '../../webgpu/gpu_test.js';
+
+export const g = makeTestGroup(GPUTest);
+
+// Helper to exhaust VRAM until there is less than 64 MB of capacity. Returns
+// an opaque closure which can be called to free the allocated resources later.
+const exhaustVramUntilUnder64MB = async (device: GPUDevice) => {
```

Comment:
This helper could be useful in other stress tests (or if at some point we decide to split this file into multiple), so this could go in another `.ts` file. If you do that, you should also add the file to the helper_index.

Also, normally we use `function` for helper functions (`async function` in this case). No real reason, but a little more readable, and consistent.

Reply:
Done